sympy/core/relational.py

```diff
@@ -1514,3 +1562,103 @@ def split_real_imag(expr):
             rv = False
         if rv is not None:
             return rv
+
+
+def _simplify_inequality_with_minmax(e, fn, cmp1, cmp2):
+    """ Simplify inequalities, i.e., ``>``, ``<``, ``>=``, ``<=``, that
+    contains a :class:`~.Min` and/or :class:`~.Max` functions.
+
+    Written so that only ``>`` and ``>=``are supported, but it is possible to
+    ``<`` and ``<=`` expressions by using ``.negated`` before and after simplifying.
+    """
+    # Simplify expressions of the type Gt(x, Min/Max(..., x, ...))
+    from sympy.functions.elementary.miscellaneous import Min, Max
+    if e.has(Min, Max):
```

Comment:
```suggestion
    if e.gts != e.lhs:
        return _simplify_inequality_with_minmax(e.negated, fn, cmp1, cmp2).negated
    if e.has(Min, Max):
```

Reply:
It seems like handling the negation is not a necessary burden to put on the calling routine.